macros/schema_tests/unique_combination_of_columns.sql

```diff
@@ -1,6 +1,7 @@
 {% macro test_unique_combination_of_columns(model, quote_columns = false) %}
 
 {%- set columns = kwargs.get('combination_of_columns', kwargs.get('arg')) %}
+{%- set where = kwargs.get('where', kwargs.get('arg')) %}
```

Comment:
ok so this one is on me — we don't actually need this line (or the above one tbh).

This line was because I was copying syntax from dbt core that handled the "v1" yaml syntax, which we haven't supported for a long time now!

Can you:
1. Remove lines 3 & 4
2. Add `where` as an arg in line 1 (`where=None`)

Reply:
Sure.